app.js

```diff
@@ -0,0 +1,15 @@
+const express = require("express");
+var cors = require('cors')
+const app     = express();
+app.use(cors())
```

Comment:
Awesome, Allow CORS from localhost, and jscms.test right now. 
Use an array called whitelisted = [] to specify allowed CORS

Reply:
Added!